src/Documentation/doc/locales/zh-CN/help/getting_started.md

```diff
@@ -0,0 +1,766 @@
+# 程序员初学者入门指南
+
+_注意_：本指南中的[脚本](../basic/scripts.md)和策略并不一定是最佳或全面的。
+本指南旨在帮助编程知识较少的用户在游戏初期体验 Bitburner。
+
+如果你对游戏，尤其是编程和脚本方面感到困惑或无从下手，那么这份指南非常适合你！
+
+## 简介
+
+Bitburner 是一款以赛博朋克为主题的增量 RPG。
+你可以通过提高[属性](../basic/stats.md)、赚钱以及提高你在现实中的编程能力来推进游戏进度。
+在达到某些条件后，你将收到来自游戏中某些[派系](../basic/factions.md)的邀请
+加入[派系](../basic/factions.md)并为其工作将会解锁各种[增强](../basic/augmentations.md)
+购买并“安装”后，它们会为[属性](../basic/stats.md)和其他能力提供持久的加成。与派系合作并安装增强是在 Bitburner 中取得进步的基本步骤。
+
+游戏有一个开放且简洁的故事线，你可以通过多种方式来实现目标。
+由于本指南是作为 Bitburner 的基础介绍而编写的，因此不会包含游戏的所有可用内容和剧情。
+
+## 开始
+
+我假设你在刚开始游戏时按入门教程进行了操作。
+在入门教程中，你创建了一个[脚本](../basic/scripts.md)调用 `n00dles.js` 并在服务器 `n00dles` 上运行它。
+现在，我们将终止此[脚本](../basic/scripts.md)。有两种方法可以做到这一点：
+
+- 前往终端并输入：`kill n00dles.js`
+- 前往 `活动的脚本 \ Active Scripts` 页面 (快捷键 Alt + s) 并点击 `n00dles.js` 的 `终止` 按钮。
+
+如果你跳过了入门教程，那么就无需理会上述内容。
+相反，前往 `黑客网 \ Hacknet Nodes` 页面 (快捷键 Alt + h) 并购买[黑客网节点](../basic/hacknet_nodes.md)来获取一些挂机收益。
+
+## 创建你的第一个脚本
+
+接下来，我们将会创建一个可以在游戏的早期使用的通用[黑客](../basic/hacking.md)[脚本](../basic/scripts.md)(如果你愿意，也可以一直用下去)。
+
+在我们写[脚本](../basic/scripts.md)前，要先熟悉以下这些内容：
+
+- `黑客攻击`
```

Comment:
This is an interesting point. For basically all the docs, `code block text` is used for stuff that appears literally, and thus it shouldn't be translated (since it won't be in-game). However, these two are general concepts as opposed to literal terms that appear in-game. It's potentially an argument that they shouldn't be formatted like this at all.

Reply:
Yes, after some consideration, I don't think any functions, instructions, codes or other unique concepts etc. should be translated, and for the UI related text, since it's not translated elsewhere, I've kept the original content. But for the two generic concepts, they are translated elsewhere, so I think they should be translated as well.
As to whether it should be included in the code block, allow me to reserve my opinion, after all I don't have any good ideas.